django/db/backends/mysql/features.py

```diff
@@ -53,6 +53,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supported_explain_formats = {'JSON', 'TEXT', 'TRADITIONAL'}
     # Neither MySQL nor MariaDB support partial indexes.
     supports_partial_indexes = False
+    can_introspect_jsonfield = True
```

Comment:
This shouldn't be `True` in all cases you cannot introspect `JSONField`'s on MySQL, you cannot introspect `JSONField` on MariaDB < 10.2.21, etc. I would use
```python
@cached_property
def can_introspect_jsonfield(self):
    return self.supports_json_field and self.can_introspect_check_constraints
```